plugin.json

```diff
@@ -146,6 +146,12 @@
             "type": "bool",
             "help_text": "When true, the plugin will not enable any sync msg infrastructure.",
             "default": false
+        },{
+            "key": "useSharedChannels",
```

Comment:
@jespino @lieut-data we should consider renaming this to something like "Reliable Message Delivery".  "Shared Channels" doesn't have much meaning in the MS Teams context, and is potentially confusing.

Reply:
This is a temporary setting, but yes, we can change it.

Reply:
Works, for me, any name suggestion?